android/app/build.gradle

```
@@ -9,47 +9,50 @@ apply plugin: 'com.android.application'
 
 // Set up Android app project.
 android {
-	compileSdkVersion 33
+	ndkVersion project.ext.tiNdkVersion
+	//namespace 'com.titanium.test'
 	defaultConfig {
 		applicationId 'com.titanium.test'
-		minSdkVersion 21
+		compileSdk 33
+		minSdkVersion 26
 		targetSdkVersion 33
 		versionCode 1
 		versionName '1.0'
 		manifestPlaceholders = project.ext.tiManifestPlaceholders
 		manifestPlaceholders.put('localApplicationId', applicationId) // Legacy placeholder old AARs sometimes use.
 	}
-	lintOptions {
-		checkReleaseBuilds false
-	}
 	sourceSets {
 		main {
 			assets.srcDirs = [
-				'src/main/assets',
-				"${projectDir}/../titanium/build/outputs/ti-assets"
+					'src/main/assets',
+					"${projectDir}/../titanium/build/outputs/ti-assets"
```

Are these indentations intentional?

```suggestion
				'src/main/assets',
				"${projectDir}/../titanium/build/outputs/ti-assets"
```